src/CMakeLists.txt

```diff
@@ -103,10 +111,18 @@ endif()
 
 
 option(ENABLE_GLES "Use OpenGL ES instead of OpenGL" FALSE)
-mark_as_advanced(ENABLE_GLES)
+set(GLES_VERSION "2" CACHE STRING "Specify OpenGL ES Version (1 or 2)")
```

Comment:
Making this a binary choice doesn't make sense either, Irrlicht can be built with support for both.

For several reasons I'd argue hardcoding GLES2 makes sense and should be kept:
* GLES1 is inferior and legacy (no shaders, which we'll only be using more frequently)
* GLES2 is available everywhere
* Recent systems may even be missing GLES1

Reply:
I mean, that is why I made it an option, though a 3rd choice to link with both might be a good idea. If using static Irrlicht with GLES1 it must be linked with as a transitive dependency, and when I tested it GLES2 shaders didn't like the PinePhone, but some changes have been merged since I tested it. What should the options be? ``1, 2, BOTH``?

Reply:
I made it ``1``, ``2``, or ``BOTH``.

Reply:
> when I tested it GLES2 shaders didn't like the PinePhone

Be more specific here. Did you copy the files from `<irrlicht>/media/Shaders` to `client/shaders/Irrlicht`?

Reply:
Yeah, I copied it.

Reply:
I spun this out into its own PR: #11424.